### ISSUE TITLE
'Operation' object has no attribute '_unconditional_update' when using pre-trained model that used batchnorm

### ISSUE THREAD

**Issue by llealgt** [tensorflow/tensorflow#43552]
When trying to use tf.keras.applications that use bacthnorm(like mobilenet or inceptionv3) i'm getting:
`'Operation' object has no attribute '_unconditional_update'`
Other keras.applications that do not use batch-norm(like vgg) work fine with the same code.

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): debian  8
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): conda
- TensorFlow version (use command below): 2.3
- Python version: 3.8.3
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.1
- GPU model and memory: Nvidia titan xp

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
When trying to use tf.keras.applications that use bacthnorm(like mobilenet or inceptionv3) i'm getting:
`'Operation' object has no attribute '_unconditional_update'` when running fit
Other keras.applications that do not use batch-norm(like vgg) work fine with the same code.
**Describe the expected behavior**
to be able to run fit on keras.applications models

**Standalone code to reproduce the issue**
use applications model: model_base = keras.applications.InceptionV3(input_shape=INPUT_SIZE,weights = "imagenet",include_top=False)
add fully connected layers to it and then call fit()

**Comment by ravikyram** [comment]
@llealgt 

Can you please share colab link or simple standalone code with supporting files to reproduce the issue in our environment.It helps us in localizing the issue faster.Thanks!

**Comment by llealgt** [comment]
I tried to isolate it,i found that this happens when eager mode is disabled with tf.compat.v1.disable_eager_execution() , i disable it because i need to export a .pb file for deployment and export fails if eager is enabled, so it seems using pre-trained model(for transfer learning) with batch norm is mutually exclusive with exporting .pb file for deploy: i can do transfer learning but not deploy it, or i can deploy it but not with pre-trained model that uses batch-norm

**Comment by jvishnuvardhan** [comment]
@llealgt Can you please share a simple standalone code to reproduce the error? Thanks!
